.github/workflows/fingertips-workflow.yml

```
@@ -583,6 +583,21 @@ jobs:
             name        = "APPLICATIONINSIGHTS_CONNECTION_STRING"
             secret_name = "fingertips-api-application-insights"
             value       = null
+          },
+          {
+            name        = "UPLOAD_STORAGE_ACCOUNT_URI"
+            secret_name = "fingertips-upload-storage-uri2"
```

I think we want to change this back now:
```suggestion
            secret_name = "fingertips-upload-storage-uri"
```